src/pluto/format/jpeg.cr

```diff
@@ -45,7 +45,7 @@ module Pluto::Format::JPEG
       from_jpeg(io.getb_to_end)
     end
 
-    private def self.check(handle, code)
+    protected def self.check(handle, code)
```

Comment:
Visibility needed to change since instance level code needed to call class level code.

I also realized that, while it didn't happen with webp or jpeg, future formats might trigger another `check` method to be defined that may collide with name and argument types of these. Should we rename them to be a bit more specific like `check_jpeg` or `check_webp` to avoid that hypothetical collision? Not a big concern right now, obviously.

Reply:
I would rename them